src/Main.java

```diff
@@ -0,0 +1,63 @@
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        ArrayList<MatchStats> listOfMatch = new ArrayList<>();
```

Comment:
Bacha na konzistenciu namingu, ja by som o zmenil treba takto:
MatchStats -> Match (alebo zmenit Team -> TeamStats)
listOfMatch -> matchList

namesOfTeam a listOfTeam mi pridu zbytocne, vid koment nizsie

Reply:
Opravil jsem nazvy objektu, aby to melo lepsi naming convention, ale nazvy trid se mi zdaji vypovidajici
Team - udaje o tymu
MatchStats - Statistiky zapasu
MatchAnalyzer - jsem vytvoril na tvoje doporuceni
TeamAnalyzer - prochazeni listu tymu a vypsani pozadovaneho

Reply:
Triedy Team a MatchStats reprezentuju velmi podobne entity. Udaje o zapasu, udaje o tymu. Stale si myslim, ze dvojice (Team, Match), alebo (TeamStats, MatchStats) by bolo vhodnejsie.